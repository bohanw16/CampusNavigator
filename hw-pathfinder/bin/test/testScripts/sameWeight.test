# test a graph that has two paths of equal weight

CreateGraph g

AddNode g n1
AddNode g n2
AddNode g n3
AddNode g n4

AddEdge g n1 n2 1.111
AddEdge g n1 n3 2.222
AddEdge g n2 n4 2.222
AddEdge g n3 n4 1.111

FindPath g n1 n4