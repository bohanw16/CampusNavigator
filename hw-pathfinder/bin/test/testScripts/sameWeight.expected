# test a graph that has two paths of equal weight

created graph g

added node n1 to g
added node n2 to g
added node n3 to g
added node n4 to g

added edge 1.111 from n1 to n2 in g
added edge 2.222 from n1 to n3 in g
added edge 2.222 from n2 to n4 in g
added edge 1.111 from n3 to n4 in g

path from n1 to n4:
n1 to n2 with weight 1.111
n2 to n4 with weight 2.222
total cost: 3.333